pyretailscience/plots/period_on_period.py

```diff
@@ -78,10 +78,11 @@ def plot(
         matplotlib.axes.Axes: The matplotlib Axes object with the completed plot.
 
     Raises:
-        ValueError: If `periods` is an empty list.
+        ValueError: The 'periods' list must contain at least two (start, end) tuples for comparison.
     """
-    if not periods:
-        raise ValueError("The 'periods' list must contain at least one (start, end) tuple.")
+    period_length = 2
+    if len(periods) < period_length:
+        raise ValueError("The 'periods' list must contain at least two (start, end) tuples for comparison.")
```

Comment:
Can you call this ˋmin_period_lengthˋ please. I think that's more descriptive
```suggestion
    min_period_length = 2
    if len(periods) < min_period_length:
        raise ValueError("The 'periods' list must contain at least two (start, end) tuples for comparison.")
```